libs/utils/utils.js

```diff
@@ -419,6 +419,40 @@ export function decorateSVG(a) {
     return a;
   }
 }
+function hasUrlWithPipe(str) {
+  const regex = /^https?:\/\/.*\|.*/;
+  return regex.test(str);
+}
+
+export function decorateImageLinks(a) {
+  const images = a.querySelectorAll('img');
+  if (!images.length) return a;
+  [...images].forEach((img) => {
+    // Check if the image has an alt tag with a url & pipe - 'https... |'
+    if (!(hasUrlWithPipe(img.alt))) return a;
+    try {
+      // //Mine for URL and alt text
+      const splitText = img.alt.split('|');
+      const textUrl = new URL(splitText.shift().trim());
+      const altText = splitText.join('|').trim();
+      if (altText) img.alt = altText;
+
+      const pic = img.closest('picture');
+      const picParent = pic.parentElement;
+      const aTag = createTag('a', { href: textUrl, class: 'desc-link' }, pic);
```

Comment:
Would it make more sense for the class to be `image-link`?  Not sure how "description" relates here.

Reply:
`desc-` was because this link is derived from the image description. I like `image-link` better though. 👍